### DISCUSSION TITLE
How do I enter a screen capture mode ?

### DISCUSSION THREAD

**Discussion by begrob** [pubpub-zz/ppInk#198] [General]
I must be missing something obvious, but once I have made my annotations on the screen, how do I enter a screen capture mode so that I can select an area and save it as a picture ? thanks

**Comment by pubpub-zz** [comment]
use the camera icon or the shortcut K, the screen will be slighlty greyed with a cross hair cursor to select the area. a short click takes a snapshot of the whole screen.

**Reply by pubpub-zz** [reply]
@begrob,
Is is OK ?

**Reply by begrob** [reply]
Hi, yes that worked, thanks.
Is there a way to disable the floating icon on the desktop, and only have the tray icon ? Didn't see anything like it in the config.ini, and didn't find any "preferences/options" on the toolbar that may have that option.
thanks !

**Reply by begrob** [reply]
Haha just as I wrote the answer above, I tried right-click-and-drag the floating icon on the desktop, and that worked :) so I dragged it out of the way, good enough for now. An option to hide it might be useful for other users too though.

**Reply by pubpub-zz** [reply]
I does exist 😊:
![image](https://user-images.githubusercontent.com/4083478/188260992-0d67f679-e2a6-4f46-ac58-e372cd272e07.png)

the checkbox is to hide/show it and the button to save the current (but you will still have to "save to files" the setting 


do not hesitate to star the project if you like it😉
